source/Halibut.TestUtils.CompatBinary.Base/EchoService.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Halibut.TestUtils.SampleProgram.Base
+{
+    public class EchoService : IEchoService
+    {
+        public string SayHello(string name)
+        {
+            return name;
+        }
+
+        public bool Crash()
+        {
+            throw new Exception();
+        }
+
+        public static Action OnLongRunningOperation { get; set; }
```

Comment:
Not static

Reply:
Removed since it can't be used across the process boundary.